get-config-state.js

```diff
@@ -40,6 +40,10 @@ function getConfigState(dirname, opts) {
         dc && NODE_ENV ?
             join(configFolder, NODE_ENV + '.' + dc.datacenter + '.json') :
             null,
+        // load ./config/secrets-NODE_ENV.json
+        NODE_ENV ? join(configFolder, 'secrets-' + NODE_ENV + '.json') : null,
```

Comment:
It's useful to allow passing in environments. Especially ability to override secrets in production level.